includes/class-wsuwp-people-post-type.php

```diff
@@ -337,6 +337,7 @@ public function admin_enqueue_scripts( $hook_suffix ) {
 			$profile_vars = array(
 				'nid_nonce' => wp_create_nonce( 'wsu-people-nid-lookup' ),
 				'post_id' => $post->ID,
+				'request_from' => ( apply_filters( 'wsuwp_people_display', true ) ) ? 'people' : 'ad',
```

Comment:
What do you think about this being something like `apply_filters( 'wsuwp_people_source', 'people' );` instead? I can see wanting to reuse a bunch of the front end stuff, so I'm not sure if tying this to whether the display functionality is turned on makes sense.

I'm also wondering if `people` / `ad` is right, or if it should be something like `rest` / `ad`.